### PR TITLE
fix(arc-1784): fix margin + fix filter + fix safari

### PR DESCRIPTION
**Pull request by WardVerc1**
image

Ticket: https://meemoo.atlassian.net/browse/ARC-1784

### CODE REVIEW

File: src/styles/pages/_object-detail.scss [hunk 1]
```diff
@@ -38,6 +43,12 @@ $metadata-collapsed-width: 36vw;
 			top: $spacer-lg;
 			left: $spacer-lg;
 		}
+
+		// `include respond-at(md)` -fix for safari
+		@media screen and (width >= 768px) {
+			top: $spacer-lg;
```
Comment (SHDBramVandenbussche) [inline src/styles/pages/_object-detail.scss:49]: Use 48rem instead of 768px

File: src/styles/pages/_object-detail.scss [hunk 2]
```diff
@@ -88,6 +99,11 @@ $metadata-collapsed-width: 36vw;
 		@include respond-at(md) {
 			display: none;
 		}
+
+		// `include respond-at(md)` -fix for safari
+		@media screen and (width >= 768px) {
```
Comment (SHDBramVandenbussche) [inline src/styles/pages/_object-detail.scss:104]: Same comment as above

File: src/styles/pages/_object-detail.scss [hunk 3]
```diff
@@ -345,6 +361,21 @@ $metadata-collapsed-width: 36vw;
...
+
+		.p-object-detail__video {
+			display: grid;
+			grid-template-rows: 1fr auto;
+			margin-top: 0.1rem; // Prevent wrong alignment (https://github.com/viaacode/hetarchief-client/pull/756#discussion_r1151752068)
+		}
+	}
+
+	// `include respond-at(md)` -fix for safari
+	@media screen and (width >= 768px) {
```
Comment (SHDBramVandenbussche) [inline src/styles/pages/_object-detail.scss:375]: Same comment as above

File: src/styles/pages/_object-detail.scss [hunk 4]
```diff
@@ -372,6 +403,11 @@ $metadata-collapsed-width: 36vw;
 		@include respond-at(md) {
 			visibility: visible;
 		}
+
+		// `include respond-at(md)` -fix for safari
+		@media screen and (width >= 768px) {
```
Comment (SHDBramVandenbussche) [inline src/styles/pages/_object-detail.scss:408]: Same comment as above

**Review by SHDBramVandenbussche — APPROVED** [review]

**Review by SHDBramVandenbussche — APPROVED** [review]

**Review by bertyhell — APPROVED** [review]

**Pull request merged**